war/pom.xml

```diff
@@ -736,7 +740,7 @@ THE SOFTWARE.
                   <goal>gulp</goal>
                 </goals>
                 <configuration>
-                  <arguments>lint bundle</arguments>
+                  <arguments>jshint bundle</arguments>
```

Comment:
@oleg-nenashev We are using the task `jshint` since js-builder `0.0.40` provides this one. When we upgrade to 0.0.51 the name of this task will be different because has been renamed.